src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,41 @@
+.stars {
+  display: flex;
+  margin-bottom: 24px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg) no-repeat;
+    padding-right: 4px;
+  }
+
+  &--1 {
+    .stars__star:nth-child(-n+1) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--2 {
+    .stars__star:nth-child(-n+2) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--3 {
+    .stars__star:nth-child(-n+3) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--4 {
+    .stars__star:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &--5 {
+    .stars__star:nth-child(-n+5) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
why don't you just took this implementation from `layout_stars`? you don't really need this scss nesting, can just:
```scss
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url(...images/star-active.svg);
}
```